### DISCUSSION TITLE
Add tooltip on column header

### DISCUSSION THREAD

**Discussion by eyperrone** [l-lin/angular-datatables#1735] [Q&A] [answered]
Hi
i actually work with Angular 11, NgBootstrap and angular-datatables.
I have a problem when  I trying to add a bootsrap tooltip on a column header. It's not working (probally the datatables js prevent this flow).
If i put the tooltip outside the table it correctly work.

Any solutions?

Thanks!

**Comment by shanmukhateja** [comment, marked as answer]
Hi,

You can try something like:

https://datatables.net/forums/discussion/70962/titleattr-for-column-headers

**Reply by eyperrone** [reply]
Thanks!
